gramps/plugins/webreport/common.py

```diff
@@ -646,7 +647,7 @@ def __init__(self, rlocale):
                     super().addLabels(loc)
 
 else:
-    AlphabeticIndex = localAlphabeticIndex
+    AlphabeticIndex = localAlphabeticIndex  # type: ignore
```

Comment:
mypy allows a `# type: ignore[$reason-code]` syntax as well where `$reason-code` is the code (readable machine name) of the specific violation you want to ignore. Doing that, in combination with enabling `warn_unused_ignores` in your favorite mypy configuration file means that you won't end up accidentally ignoring the wrong things as the codebase changes over time.